external/storm-kafka-client/src/test/java/org/apache/storm/kafka/spout/builders/SingleTopicKafkaSpoutConfiguration.java

```diff
@@ -41,31 +41,22 @@ public static Config getConfig() {
 
     public static StormTopology getTopologyKafkaSpout(int port) {
         final TopologyBuilder tp = new TopologyBuilder();
-        tp.setSpout("kafka_spout", new KafkaSpout<>(getKafkaSpoutConfig(port)), 1);
+        tp.setSpout("kafka_spout", new KafkaSpout<>(getKafkaSpoutConfigBuilder(port).build()), 1);
```

Comment:
The SingleTopicKafkaSpoutConfiguration class was beginning to turn into a mess of telescoping constructors, and it was a bit silly given that we already have a nice builder interface underlying that class.